packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActions.base.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { BaseComponent, classNamesFunction } from '../../Utilities';
+import { Icon } from '../../Icon';
+import { IDocumentCardActionsProps, IDocumentCardActionsStyleProps, IDocumentCardActionsStyles } from './DocumentCardActions.types';
+import { IconButton } from '../../Button';
+import { IProcessedStyleSet } from '../../Styling';
+
+const getClassNames = classNamesFunction<IDocumentCardActionsStyleProps, IDocumentCardActionsStyles>();
+
+export class DocumentCardActionsBase extends BaseComponent<IDocumentCardActionsProps, any> {
```

Comment:
nit: Seems an overkill to have this a class component vs a functional component

Reply:
Looking around the code base I'm seeing class components everywhere, so I'll leave these as is.